ds-collector/ds-collector

```diff
@@ -1090,7 +1086,17 @@ then
   jmxPassword=$(escape_special_chars "$jmxPassword")
 fi
 
-if [ $testMode ]; then
+
+if [ $debug ] ; then
+  set -x
+fi
+if [ $verbose ] ; then
+  bastion_checks
+  logFile="${baseDir}/ds-collector-$(date +'%Y-%m-%d-%H-%M-%s').log"
+  echo "debugging to ${logFile}"
+  ( exec $0 ${@/-v/-x} )  2>&1  | sed "s/${jmxPassword:-'\*\*\*\*'}/****/g" | sed "s/${cqlshPassword:-'\*\*\*\*'}/****/g" | tee "${logFile}"
```

Comment:
this is the crux of the new verbose/logging patch

Reply:
We need the branch name fix overlaying to be able to test this - and I don't see any of the make files adjusted to use the debug / verbose

Reply:
i will add a test that uses it…